index.bs

```diff
@@ -4278,6 +4875,22 @@ partial interface MLGraphBuilder {
                                [EnforceRange] unsigned long hiddenSize,
                                optional MLLstmCellOptions options = {});
 };
+
+dictionary MLLstmCellSupportLimits {
+  MLSupportLimits input;
+  MLSupportLimits weight;
+  MLSupportLimits recurrentWeight;
+  MLSupportLimits hiddenState;
+  MLSupportLimits cellState;
+  MLSupportLimits bias;
+  MLSupportLimits recurrentBias;
+  MLSupportLimits peepholeWeight;
+  MLSupportLimits output;
```

Comment:
It should be outputs because lstmCell also returns a sequence of operands. Spotted in Chromium CL review: https://chromium-review.googlesource.com/c/chromium/src/+/5821811/comment/43ea6f39_e85e92e4/